Ratios/CalculatorViewController.swift

```diff
@@ -114,6 +165,14 @@ class CalculatorViewController: UIViewController {
 
         self.stackViewWidthConstraint?.isActive = true
     }
+    
+    func updateBackgroundColor(_ ratio: Int) {
+        UIView.animate(withDuration: 0.15) {
```

Comment:
Lets move the `UIView.animate` to the call site, or pass in the duration so we can choose whether we want the colour to be animated. Perhaps on initial launch we'd might want to just go straight to the colour without animation.